containers/html/lists.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/containers/css/lists.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="list-container">
+        <div class="ordered-list">
+            <ol>
+                <li>item1</li>
+                <li>item2</li>
+                <li>item3</li>
+            </ol>
+        </div>
+        <div class="unordered-list">
```

Comment:
Make a general div named like `example-div` so that you can use it anywhere and import it in main css file.